### ISSUE TITLE
Ruby ApiError bug

### ISSUE THREAD

**Issue by TheStu** [mailchimp/mailchimp-client-lib-codegen#255]
### What issue did you have?

Does this repo take PRs?

https://github.com/mailchimp/mailchimp-marketing-ruby/blob/af7eb66cf51424b6a9db38db8a9bdf31e9b3764a/lib/MailchimpMarketing/api_client.rb#L78

`fail ApiError.new(:status => res.status, :response_body => res.body)`

`res.body` is a string but ApiError is expecting a hash. Needs to be changed to:

`fail ApiError.new(:status => res.status, :response_body => JSON.parse(res.body))` or something to that effect.

### Language

Ruby

**Comment by Moishe** [comment]
Hi there! This repo doesn't take PRs -- it's autogenerated from this repo: https://github.com/mailchimp/mailchimp-client-lib-codegen

I can open up a PR over there to address that and try to get it merged today (which will update this repo), though!

**Comment by TheStu** [comment]
Oh sorry when I said 'does this repo take PRs?' I was talking about this repo, not the repo I linked to.

**Comment by Moishe** [comment]
Oh sorry I totally misinterpreted! Yes this repo takes PRs and if you'd like to open one to fix this, please do!

**Comment by TheStu** [comment]
@Moishe actually I looked and I have no idea what's going on with the mustache code. If you can make a PR to fix this that would be much appreciated :)

**Comment by Moishe** [comment]
Yeah, totally, it's a little confusing! I'll take a look at it on Monday.

**Comment by Moishe** [comment]
Just a ping to say I haven't forgotten about this. This is turning into something of a yak shave about how tests get run in this repo but I'll update it as I go on.
